src/Polytopes/SubdivisionOfPoints/constructors.jl

```diff
@@ -0,0 +1,93 @@
+###############################################################################
+###############################################################################
+### Definition and constructors
+###############################################################################
+###############################################################################
+
+@doc Markdown.doc"""
+    SubdivisionOfPoints(Points::Union{Oscar.MatElem,AbstractMatrix}, Cells::IncidenceMatrix)
+
+A subdivision of points formed from points and cells made of these points.
+
+`Points` is given row-wise as representative vectors. The cells are given as an
+IncidenceMatrix, where the columns represent the points and the rows represent
+the cells. There is a `1` at position `(i,j)` if cone `i` has ray `j` as
+extremal ray, otherwise there is a `0`.
+
+# Examples
+The following is the famous "mother of all examples" (moae) non-regular
+triangulation.
+```julia-repl
+julia> moaepts = [4 0 0; 0 4 0; 0 0 4; 2 1 1; 1 2 1; 1 1 2];
+
+julia> moaeimnonreg0 = IncidenceMatrix([[4,5,6],[1,4,2],[2,4,5],[2,3,5],[3,5,6],[1,3,6],[1,4,6]]);
+
+julia> MOAE = SubdivisionOfPoints(moaepts, moaeimnonreg0);
+```
+"""
+struct SubdivisionOfPoints
+   pm_subdivision::Polymake.BigObject
+   function SubdivisionOfPoints(pm::Polymake.BigObject)
+      return new(pm)
+   end
+end
+
+@doc Markdown.doc"""
+    SubdivisionOfPoints(Points, Cells)
```

Comment:
this is the same constructor as documented above, but the one with a weight vector is not documented yet?